Helper/Data.php

```diff
@@ -43,6 +42,11 @@ class Data extends AbstractHelper
 
     const CONFIG_CC_OPEN_ONCTRACT = 'payment/reach_cc/allow_open_contract';
 
+    const DHL_PREF_TARIFFS          = "payment/reach_payment/reach_dhl/pref_tariffs";
```

Comment:
do we need a get function for this config value?

Reply:
> Let me be bit more specific: with various values available in the toOptionArray as well as corresponding 1 would correct default value be set at the beginning after page rendering? This comment is about each of qualifiesForPreferentialTariffs, pricingStrategy, clearanceMode, endUse, transportMode. Thank you!

Not 100% following. During system:upgrade, the Magento system will use 'defaults.xml' to create the database entries if they do not yet currently exist and fill the values with the default values.

When the page loads (what you seem to be asking), it will display those default values until they are changed by the user.

Reply:
Escaped the html tags to match Github markdown and this is how it looks like now:

Let me be bit more specific: with various values available in the toOptionArray as well as corresponding <field id="active">1</field> would correct default value be set at the beginning after page rendering? This comment is about each of qualifiesForPreferentialTariffs, pricingStrategy, clearanceMode, endUse, transportMode. Thank you! 

Does that help?

Reply:
Again, not following what you are asking. The XML snippet you quoted is saying that this field should not be editable unless the parent is enabled. For example, if DHL is disabled then all sub-items will be grayed out in the UX.

Reply:
If it is behaving/setting things as intended (as per your test before commit) then I am good. Thanks!